src/components/Adyen/Adyen.js

```diff
@@ -262,6 +299,13 @@ const Adyen = ({
           ...adyenConfiguration?.paymentMethodsConfiguration?.card
         },
         applepay: {
+          onClick: async resolve => {
+            if (!isCheckboxChecked('applepay')) {
+              return;
+            }
+
+            resolve();
+          },
```

Comment:
do we need async here?

Reply:
Nope, just copied it from gifting and didn't remove. Will remove now but it will be back with gifting (we update order there)